### DISCUSSION TITLE
Joint control of yaw angle and axial induction factor using FLORIS model

### DISCUSSION THREAD

**Discussion by zyla123** [NatLabRockies/floris#1048] [Q&A]
At present, I want to use the FLORIS model for joint control of yaw angle and axial induction factor. Some literature has adopted the FLORIS model, but as far as I know, the interfaces of the FLORISI model are controlled by controlling the yaw angle. If I want to not use FLORIS for joint control, what do I need to change? Or is it because of limited experience that similar code has already been developed?

**Comment by zyla123** [comment]
At present, I want to use the FLORIS model for joint control of yaw angle and axial induction factor. Some literature has adopted the FLORIS model, but as far as I know, the interfaces of the FLORISI model are controlled by controlling the yaw angle. If I want to use FLORIS for joint control, what do I need to change? Or is it because of limited experience that similar code has already been developed?

**Reply by zyla123** [reply]
Perhaps by changing the CT and CP of the input file for the wind turbine, which are related to the yaw angle and axial induction factor based on empirical formulas, I can use them as inputs for the turbine and directly run the FLORIS model? Do you still need code debugging in other areas?

**Comment by misi9170** [comment]
Hi @zyla123 ,

Please see #835, #993, etc for answers to similar questions. The way to control the axial induction of turbines in FLORIS is through derating turbines. We do not currently support both yawing and derating at the same time, although some new models being introduced (#832 , #924 ) may help with this.
